### PR TITLE
css-positioning - Lamberto Martinez

### PR DESCRIPTION
**Pull request by Lambpato**
Screenshot 2023-01-27 at 9 43 28 AM
Screenshot 2023-01-27 at 9 43 51 AM
Screenshot 2023-01-27 at 9 44 02 AM

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: relative;
```
Comment (JenBowman1) [inline css-positioning/absolute.css:30]: position should be absolute

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  top: 160px;
+}
+
+.overlay {
```
Comment (JenBowman1) [inline css-positioning/absolute.css:34]: please add position: absolute as well as top, bottom, left, and right properties here to make the gradient show up over the image!

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
Hi Lambo, just a couple small changes!

File: css-positioning/absolute.css (hunk 2 above)
Comment (Lambpato) [reply css-positioning/absolute.css:34]: i tried using top, bottom, left, and right properties, but even with the developer tools the <div .overlay> would show that it has 0x0 pixels so i went with height and width instead

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**